packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -2887,7 +2887,7 @@ describe("database test", async () => {
         const expectedEService: EService = {
           ...eservice,
           id: unsafeBrandId(writtenPayload.eservice!.id),
-          name: `${eservice.name} - clone - ${formatClonedEServiceDate(
+          name: `${eservice.name} - clone - ${formatDateAndTime(
```

Comment:
To avoid exceeding the name length limit, we apply the following logic:

- if the length of the `name + suffix` ("- clone - \<date-time\>") is <= the maximum length (60) then you can create the new name without further action
- if the length is > then the resulting string must be cut by a number of characters equal to the length of the suffix + 3.
The new name will then be constructed as follows: `trimmed string + "..." + suffix`.

Here an example:
original name: `Finalitá che va clonata nei limite previsto di 60 caratteri`
cloned name: `Finalitá che va clonata n... - cloned - 2024/05/15 11:11:11`

Reply:
Updated